src/couch_replicator_scheduler.erl

```diff
@@ -50,6 +50,7 @@
 -record(job, {
           id :: job_id(),
           rep :: #rep{},
+          continuous :: boolean(),
```

Comment:
This was mainly to make using a select pattern easier further below. 

`ets:select(?MODULE, [{#job{pid = '$1', continuous = Continuous, _='_'},` 

However, I like the idea of pulling out filtering of continuous to a separate function (as indicated below, so then this doesn't apply anymore). 

Will fix it.